saas_app/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.4'
+
+services:
+  saas_app:
+    image: saas
+    build:
+      context: .
+      dockerfile: ./Dockerfile
```

Comment:
when you move this file down into the .devcontainer dir, you just need to adjust the path to `../Dockerfile` or something like that (hopefully that works)

Reply:
Done.

Reply:
Thank you!